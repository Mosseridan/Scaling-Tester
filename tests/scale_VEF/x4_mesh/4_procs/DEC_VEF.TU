Statistiques de post resolution

Temps total                       6.55362

