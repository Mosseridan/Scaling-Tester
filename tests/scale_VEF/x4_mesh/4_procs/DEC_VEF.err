Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VEF
feenableexcept enabled.
Data file : DEC_VEF
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VEF/x4_mesh/4_procs
Nom du cas DEC_VEF
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
Reading of data for a Schema_euler_explicite time scheme
Reading of the maximum number of time steps
The next backup, by security, will take place after 23 hours of calculation.
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block Cavite1
Step of 3D mesh in progress... 
It was created 33% of coordxIt was created 66% of coordx
It was created 10% of coordyIt was created 20% of coordyIt was created 30% of coordyIt was created 40% of coordyIt was created 50% of coordyIt was created 60% of coordyIt was created 70% of coordyIt was created 80% of coordyIt was created 90% of coordy
It was created 33% of coordzIt was created 66% of coordz
Step of mesh ended 
Reading of the boundary paroi_a_40
End of reading of the boundary : paroi_a_40
Reading of the boundary paroi_a_20
End of reading of the boundary : paroi_a_20
Reading of the boundary paroi_a_40
End of reading of the boundary : paroi_a_40
Reading of the boundary Paroi_echange1
End of reading of the boundary : Paroi_echange1
Reading of the boundary Devant1
End of reading of the boundary : Devant1
Reading of the boundary Derriere1
End of reading of the boundary : Derriere1
Reading of the block Cavite2
Step of 3D mesh in progress... 
It was created 14% of coordxIt was created 28% of coordxIt was created 42% of coordxIt was created 57% of coordxIt was created 71% of coordxIt was created 85% of coordx
It was created 33% of coordyIt was created 66% of coordy
It was created 33% of coordzIt was created 66% of coordz
Step of mesh ended 
Reading of the boundary Paroi_echange1
End of reading of the boundary : Paroi_echange1
Reading of the boundary paroi_a_40
End of reading of the boundary : paroi_a_40
Reading of the boundary paroi_a_20
End of reading of the boundary : paroi_a_20
Reading of the boundary Devant2
End of reading of the boundary : Devant2
Reading of the boundary Derriere2
End of reading of the boundary : Derriere2
16 double nodes were found 
Compressing objects of type Zone in progress... 
agglomeration of Zones Cavite1 and Cavite2
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom_solide
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block Cavite3
Step of 3D mesh in progress... 
It was created 14% of coordxIt was created 28% of coordxIt was created 42% of coordxIt was created 57% of coordxIt was created 71% of coordxIt was created 85% of coordx
It was created 14% of coordyIt was created 28% of coordyIt was created 42% of coordyIt was created 57% of coordyIt was created 71% of coordyIt was created 85% of coordy
It was created 33% of coordzIt was created 66% of coordz
Step of mesh ended 
Reading of the boundary Paroi_echange2
End of reading of the boundary : Paroi_echange2
Reading of the boundary Entree
End of reading of the boundary : Entree
Reading of the boundary Paroi
End of reading of the boundary : Paroi
Reading of the boundary Paroi_echange2
End of reading of the boundary : Paroi_echange2
Reading of the boundary Sortie
End of reading of the boundary : Sortie
Reading of the boundary Paroi
End of reading of the boundary : Paroi
Reading of the boundary Devant3
End of reading of the boundary : Devant3
Reading of the boundary Derriere3
End of reading of the boundary : Derriere3
Compressing objects of type Zone in progress... 
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom_fluide
Execution of the interpreter Tetraedriser_homogene_compact
Association of the domain on which the interpreter applies Tetraedriser_homogene_compact
 NB ELEM : 153 NB NODE : 288
Construction of the OctreeRoot OK 
We have split the cubes...
Splitting of the boundaries
Splitting of the connectors
Splitting of the internal faces
END of Tetraedriser_homogene_compact...
  1 NbElem=3672  NbNod=1011
Execution of the interpreter Tetraedriser_homogene_compact
Association of the domain on which the interpreter applies Tetraedriser_homogene_compact
 NB ELEM : 147 NB NODE : 256
Construction of the OctreeRoot OK 
We have split the cubes...
Splitting of the boundaries
Splitting of the connectors
Splitting of the internal faces
END of Tetraedriser_homogene_compact...
  1 NbElem=3528  NbNod=935
Execution of the interpreter Raffiner_anisotrope
Refinement of each element into 4 new tetrahedra... 
Association of the domain on which the interpreter applies Raffiner_anisotrope
We have split the tetrahedra ...
END of Raffiner_anisotrope...
  1 NbElem=14688  NbNod=4683
Refinement... OK
Execution of the interpreter Raffiner_anisotrope
Refinement of each element into 4 new tetrahedra... 
Association of the domain on which the interpreter applies Raffiner_anisotrope
We have split the tetrahedra ...
END of Raffiner_anisotrope...
  1 NbElem=14112  NbNod=4463
Refinement... OK
Decouper : Splitting of a domain
 Domain name to split : dom_solide
==============================================
The extreme coordinates of the domain dom_solide are:
x is between 0 and 1
y is between 0 and 1
z is between 0 and 1
==============================================
==============================================
The volume cells of the domain dom_solide are:
sum(volume cells)= 0.51
mean(volume cells)= 3.47222e-05
min(volume cells)= 3.47222e-05
max(volume cells)= 3.47222e-05
==============================================
domaine = dom_solide
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 166
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 2
The partitioner has generated 4 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 4 parts:
 Construction of part number 0
  Number of nodes    : 1209
  Number of elements : 3671
  Number of joints   : 2
  Joint 0 PeVoisin 2 NbSommets 42 NbFaces 63 NbElemDist 1038
  Joint 1 PeVoisin 3 NbSommets 40 NbFaces 56 NbElemDist 882
               Total:    NbSommets 82 NbFaces 119 NbElemDist 1920
Writing part 0 into the binary file DOM1_0000.Zones
Writing of 1209 nodes.
Writing of 3671 elements.
 Construction of part number 1
  Number of nodes    : 1193
  Number of elements : 3672
  Number of joints   : 1
  Joint 0 PeVoisin 2 NbSommets 33 NbFaces 47 NbElemDist 660
               Total:    NbSommets 33 NbFaces 47 NbElemDist 660
Writing part 1 into the binary file DOM1_0001.Zones
Writing of 1193 nodes.
Writing of 3672 elements.
 Construction of part number 2
  Number of nodes    : 1199
  Number of elements : 3673
  Number of joints   : 2
  Joint 0 PeVoisin 0 NbSommets 42 NbFaces 63 NbElemDist 875
  Joint 1 PeVoisin 1 NbSommets 33 NbFaces 47 NbElemDist 740
               Total:    NbSommets 75 NbFaces 110 NbElemDist 1615
Writing part 2 into the binary file DOM1_0002.Zones
Writing of 1199 nodes.
Writing of 3673 elements.
 Construction of part number 3
  Number of nodes    : 1197
  Number of elements : 3672
  Number of joints   : 1
  Joint 0 PeVoisin 0 NbSommets 40 NbFaces 56 NbElemDist 791
               Total:    NbSommets 40 NbFaces 56 NbElemDist 791
Writing part 3 into the binary file DOM1_0003.Zones
Writing of 1197 nodes.
Writing of 3672 elements.
End of the interpreter Decouper
Decouper : Splitting of a domain
 Domain name to split : dom_fluide
==============================================
The extreme coordinates of the domain dom_fluide are:
x is between 0.3 and 1
y is between 0.3 and 1
z is between 0 and 1
==============================================
==============================================
The volume cells of the domain dom_fluide are:
sum(volume cells)= 0.49
mean(volume cells)= 3.47222e-05
min(volume cells)= 3.47222e-05
max(volume cells)= 3.47222e-05
==============================================
domaine = dom_fluide
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 218
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 No correction to be made
The partitioner has generated 4 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 4 parts:
 Construction of part number 0
  Number of nodes    : 1153
  Number of elements : 3528
  Number of joints   : 3
  Joint 0 PeVoisin 1 NbSommets 34 NbFaces 46 NbElemDist 1775
  Joint 1 PeVoisin 2 NbSommets 34 NbFaces 47 NbElemDist 1746
  Joint 2 PeVoisin 3 NbSommets 14 NbFaces 13 NbElemDist 1065
               Total:    NbSommets 82 NbFaces 106 NbElemDist 4586
Writing part 0 into the binary file DOM2_0000.Zones
Writing of 1153 nodes.
Writing of 3528 elements.
 Construction of part number 1
 Adding of a new joint : 2
  Number of nodes    : 1151
  Number of elements : 3528
  Number of joints   : 3
  Joint 0 PeVoisin 0 NbSommets 34 NbFaces 46 NbElemDist 1456
  Joint 1 PeVoisin 2 NbSommets 0 NbFaces 0 NbElemDist 157
  Joint 2 PeVoisin 3 NbSommets 41 NbFaces 58 NbElemDist 1901
               Total:    NbSommets 75 NbFaces 104 NbElemDist 3514
Writing part 1 into the binary file DOM2_0001.Zones
Writing of 1151 nodes.
Writing of 3528 elements.
 Construction of part number 2
 Adding of a new joint : 1
  Number of nodes    : 1154
  Number of elements : 3528
  Number of joints   : 3
  Joint 0 PeVoisin 0 NbSommets 34 NbFaces 47 NbElemDist 1622
  Joint 1 PeVoisin 1 NbSommets 0 NbFaces 0 NbElemDist 155
  Joint 2 PeVoisin 3 NbSommets 39 NbFaces 54 NbElemDist 1804
               Total:    NbSommets 73 NbFaces 101 NbElemDist 3581
Writing part 2 into the binary file DOM2_0002.Zones
Writing of 1154 nodes.
Writing of 3528 elements.
 Construction of part number 3
  Number of nodes    : 1155
  Number of elements : 3528
  Number of joints   : 3
  Joint 0 PeVoisin 0 NbSommets 14 NbFaces 13 NbElemDist 682
  Joint 1 PeVoisin 1 NbSommets 41 NbFaces 58 NbElemDist 1980
  Joint 2 PeVoisin 2 NbSommets 39 NbFaces 54 NbElemDist 1983
               Total:    NbSommets 94 NbFaces 125 NbElemDist 4645
Writing part 3 into the binary file DOM2_0003.Zones
Writing of 1155 nodes.
Writing of 3528 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
