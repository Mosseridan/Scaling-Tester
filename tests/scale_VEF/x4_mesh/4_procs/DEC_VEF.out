-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF
 
Reading data file 

14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
15 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 4
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[2.50e-01]   1=[2.50e-01]   2=[2.50e-01]   3=[2.50e-01]
   Allowed maximum load imbalance: 1.001 

     14688      57864      57864 [20] [     1101:   14688 ]
      7573      40170      43634 [20] [     1101:   14688 ]
      4068      21546      34858 [20] [     1101:   14688 ]
      2190      13872      25616 [20] [     1101:   14688 ]
      1185       8510      20108 [20] [     1101:   14688 ]
       636       4874      15748 [20] [     1101:   14688 ]
       349       2764      12088 [20] [     1101:   14688 ]
       187       1518       9572 [20] [     1101:   14688 ]
        98        832       7538 [20] [     1101:   14688 ]
        52        422       6076 [20] [     1101:   14688 ]
        28        184       4820 [20] [     1101:   14688 ]
        19        120       4296 [20] [     1101:   14688 ]
      7345      28888      28888 [20] [      550:    7345 ]
      3813      20024      21824 [20] [      550:    7345 ]
      2056      10676      17384 [20] [      550:    7345 ]
      1110       6856      12800 [20] [      550:    7345 ]
       595       4160      10028 [20] [      550:    7345 ]
       321       2378       7758 [20] [      550:    7345 ]
       176       1344       5904 [20] [      550:    7345 ]
        95        734       4626 [20] [      550:    7345 ]
        51        392       3718 [20] [      550:    7345 ]
        29        210       3024 [20] [      550:    7345 ]
        17        118       2614 [20] [      550:    7345 ]
      7343      28850      28850 [20] [      550:    7343 ]
      3820      19770      21804 [20] [      550:    7343 ]
      2060      10570      17244 [20] [      550:    7343 ]
      1105       6762      12536 [20] [      550:    7343 ]
       597       4066       9776 [20] [      550:    7343 ]
       328       2332       7544 [20] [      550:    7343 ]
       183       1334       5794 [20] [      550:    7343 ]
        99        726       4514 [20] [      550:    7343 ]
        53        378       3438 [20] [      550:    7343 ]
        29        194       2662 [20] [      550:    7343 ]
        19        112       2350 [20] [      550:    7343 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.026
     Coarsening: 		   0.018
            Matching: 			   0.008
            Contract: 			   0.009
     Initial Partition: 	   0.000
     Uncoarsening: 		   0.007
          Refinement: 			   0.004
          Projection: 			   0.003
     Splitting: 		   0.001
********************************************************************

 gk_mcore statistics
           coresize:       176568         nmops:         2048  cmop:      0
        num_callocs:          347   num_hallocs:            0
       size_callocs:      1687104  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       176272   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 4
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[2.50e-01]   1=[2.50e-01]   2=[2.50e-01]   3=[2.50e-01]
   Allowed maximum load imbalance: 1.001 

     14112      55720      55720 [20] [     1058:   14112 ]
      7259      39186      42014 [20] [     1058:   14112 ]
      3899      21048      33824 [20] [     1058:   14112 ]
      2102      13540      25158 [20] [     1058:   14112 ]
      1123       8186      19788 [20] [     1058:   14112 ]
       609       4774      15504 [20] [     1058:   14112 ]
       323       2702      11878 [20] [     1058:   14112 ]
       171       1516       9328 [20] [     1058:   14112 ]
        91        828       7346 [20] [     1058:   14112 ]
        47        410       5930 [20] [     1058:   14112 ]
        25        202       4810 [20] [     1058:   14112 ]
        20        154       4550 [20] [     1058:   14112 ]
      7056      27746      27746 [20] [      529:    7056 ]
      3656      19230      20946 [20] [      529:    7056 ]
      1973      10342      16720 [20] [      529:    7056 ]
      1057       6608      12310 [20] [      529:    7056 ]
       577       3936       9690 [20] [      529:    7056 ]
       313       2266       7568 [20] [      529:    7056 ]
       170       1304       5758 [20] [      529:    7056 ]
        92        736       4520 [20] [      529:    7056 ]
        47        386       3504 [20] [      529:    7056 ]
        25        192       2756 [20] [      529:    7056 ]
        19        138       2534 [20] [      529:    7056 ]
      7056      27738      27738 [20] [      529:    7056 ]
      3671      19198      20968 [20] [      529:    7056 ]
      1974      10248      16662 [20] [      529:    7056 ]
      1069       6612      12192 [20] [      529:    7056 ]
       574       3900       9538 [20] [      529:    7056 ]
       313       2220       7344 [20] [      529:    7056 ]
       164       1218       5390 [20] [      529:    7056 ]
        88        668       4188 [20] [      529:    7056 ]
        48        372       3366 [20] [      529:    7056 ]
        26        196       2612 [20] [      529:    7056 ]
        19        134       2428 [20] [      529:    7056 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.024
     Coarsening: 		   0.015
            Matching: 			   0.007
            Contract: 			   0.007
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.007
          Refinement: 			   0.004
          Projection: 			   0.003
     Splitting: 		   0.001
********************************************************************

 gk_mcore statistics
           coresize:       169656         nmops:         2048  cmop:      0
        num_callocs:          341   num_hallocs:            0
       size_callocs:      1615704  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       169360   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

17 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 6.55362 s
